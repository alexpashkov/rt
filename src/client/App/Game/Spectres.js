import React from 'react';
import styled from 'react-emotion';
import Board from './Board/Board';

const Wrapper = styled.div`
  display: flex;
  flex-wrap: wrap;
  align-items: flex-start;
  color: ${({ theme }) => theme.colors.secondaryText};
`;

const SpectreWrapper = styled.div`
  width: 50%;
  padding: 5px;
`;

export default ({ spectres = [] }) => (
  <Wrapper>
    {spectres.length
      ? spectres.sort((a, b) => a.id - b.id).map(({ spectre, id }, i) => (
          <SpectreWrapper key={id}>
            <h2 css={`
              font-size: 1rem;
              overflow: hidden;
              text-overflow: ellipsis;
            `}>{id}</h2>
            <Board board={spectre} />
          </SpectreWrapper>
        ))
      : 'You are alone'}
  </Wrapper>
);
