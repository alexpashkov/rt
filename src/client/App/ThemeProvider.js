import React from 'react';
import { ThemeProvider as EmotionThemeProvider } from 'emotion-theming';
import theme from '../theme';

export const ThemeProvider = ({ children }) => (
  <EmotionThemeProvider theme={theme}>{children}</EmotionThemeProvider>
);

export default ThemeProvider;
