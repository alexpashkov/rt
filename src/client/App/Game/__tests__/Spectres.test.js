import React from 'react';
import renderer from 'react-test-renderer';
import { ThemeProvider } from '../../ThemeProvider';
import Spectres from '../Spectres';

describe('Spectres', () => {
  it('Renders', () => {
    const component = renderer.create(
      <ThemeProvider>
        <Spectres />
      </ThemeProvider>
    );
    expect(component.toJSON()).toMatchSnapshot();
  });
});
