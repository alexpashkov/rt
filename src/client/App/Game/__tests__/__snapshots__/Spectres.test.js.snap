// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spectres Renders empty 1`] = `
<div
  className="css-bkjqz6 e9pbz0x0"
>
  You are alone
</div>
`;

exports[`Spectres Renders with spectres 1`] = `
<div
  className="css-bkjqz6 e9pbz0x0"
>
  <div
    className="css-11ze7cv e9pbz0x1"
  >
    <div>
      <div
        className="css-ocvors e18btvrt1"
      >
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
      </div>
      <div
        className="css-ocvors e18btvrt1"
      >
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
      </div>
      <div
        className="css-ocvors e18btvrt1"
      >
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
      </div>
      <div
        className="css-ocvors e18btvrt1"
      >
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
      </div>
      <div
        className="css-ocvors e18btvrt1"
      >
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
      </div>
      <div
        className="css-ocvors e18btvrt1"
      >
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
      </div>
      <div
        className="css-ocvors e18btvrt1"
      >
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
      </div>
      <div
        className="css-ocvors e18btvrt1"
      >
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
      </div>
      <div
        className="css-ocvors e18btvrt1"
      >
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
      </div>
      <div
        className="css-ocvors e18btvrt1"
      >
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
      </div>
      <div
        className="css-ocvors e18btvrt1"
      >
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
      </div>
      <div
        className="css-ocvors e18btvrt1"
      >
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
      </div>
      <div
        className="css-ocvors e18btvrt1"
      >
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
      </div>
      <div
        className="css-ocvors e18btvrt1"
      >
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
      </div>
      <div
        className="css-ocvors e18btvrt1"
      >
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
      </div>
      <div
        className="css-ocvors e18btvrt1"
      >
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
      </div>
      <div
        className="css-ocvors e18btvrt1"
      >
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
      </div>
      <div
        className="css-ocvors e18btvrt1"
      >
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
      </div>
      <div
        className="css-ocvors e18btvrt1"
      >
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
      </div>
      <div
        className="css-ocvors e18btvrt1"
      >
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
      </div>
    </div>
  </div>
  <div
    className="css-11ze7cv e9pbz0x1"
  >
    <div>
      <div
        className="css-ocvors e18btvrt1"
      >
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
      </div>
      <div
        className="css-ocvors e18btvrt1"
      >
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
      </div>
      <div
        className="css-ocvors e18btvrt1"
      >
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
      </div>
      <div
        className="css-ocvors e18btvrt1"
      >
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
      </div>
      <div
        className="css-ocvors e18btvrt1"
      >
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
      </div>
      <div
        className="css-ocvors e18btvrt1"
      >
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
      </div>
      <div
        className="css-ocvors e18btvrt1"
      >
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
      </div>
      <div
        className="css-ocvors e18btvrt1"
      >
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
      </div>
      <div
        className="css-ocvors e18btvrt1"
      >
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
      </div>
      <div
        className="css-ocvors e18btvrt1"
      >
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
      </div>
      <div
        className="css-ocvors e18btvrt1"
      >
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
      </div>
      <div
        className="css-ocvors e18btvrt1"
      >
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
      </div>
      <div
        className="css-ocvors e18btvrt1"
      >
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
      </div>
      <div
        className="css-ocvors e18btvrt1"
      >
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
      </div>
      <div
        className="css-ocvors e18btvrt1"
      >
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
      </div>
      <div
        className="css-ocvors e18btvrt1"
      >
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
      </div>
      <div
        className="css-ocvors e18btvrt1"
      >
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
      </div>
      <div
        className="css-ocvors e18btvrt1"
      >
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
      </div>
      <div
        className="css-ocvors e18btvrt1"
      >
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
      </div>
      <div
        className="css-ocvors e18btvrt1"
      >
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
        <div
          className="css-iil78m e18btvrt0"
        />
      </div>
    </div>
  </div>
</div>
`;
